docs/rules/no-return-await.md

```diff
@@ -11,6 +11,8 @@ This rule aims to prevent a likely common performance hazard due to a lack of un
 Examples of **incorrect** code for this rule:
 
 ```js
+/*eslint no-return-await: "error"*/
```

Comment:
[no-await-in-loop](https://github.com/eslint/eslint/blob/master/docs/rules/no-await-in-loop.md) doesn't have this directive as well. 
It looks like we are having this in almost all rules, so maybe we can fix that in this PR or may be in a separate PR as well.

Reply:
Nice find! I made PR #13673 to fix that.

Reply:
Thanks